### ISSUE TITLE
Display unexplored areas as thick cloud cover/"fog of war"

### ISSUE THREAD

**Issue by Abbie5** [sleepingdragoninn/antique-atlas#26]
I think this would make it a lot easier to distinguish unexplored areas from explored but empty ones.

**Comment by sleepingdragoninn** [comment]
Great idea, thanks!

**Comment by sleepingdragoninn** [comment]
I gave this a shot, because I was interested in experimenting with how it would look - I think this cloud texture is too noisy for me to use, but I committed it anyhow for anyone who'd like to potentially expand on the concept: https://github.com/sisby-folk/antique-atlas/commit/72154e80607f1e9ef3c50d38702e247f5254e56f

![image](https://github.com/sisby-folk/antique-atlas/assets/55819817/8a7751e8-ebf6-4d3a-a001-de9ff1ef2a9c)

![image](https://github.com/sisby-folk/antique-atlas/assets/55819817/872f3de5-f262-414a-8721-e9d36e953986)

I think generally this is a reasonable idea, but will definitely need a config toggle. there's a certain believability to "I haven't drawn that part of the map yet" that might be lost, here.

**Comment by Abbie5** [comment]
That looks good, it still has the issue of edge tiles being drawn as if the neighbouring tiles are empty land tiles though. (See the false "coast" in the south in the second screenshot you posted)

Sorry, I should have mentioned this earlier.

**Comment by sleepingdragoninn** [comment]
yeah I couldn't quite get every other tile to stitch to it like I wanted - we're redoing stitching in 2.0 so we might try again after that - technically we need things like forests to not stitch, because they get cut off otherwise.

**Comment by sleepingdragoninn** [comment]
We're improving on this mildly in 1.4 by making the end void not blank - it has "static" like the end sky.
